GNUmakefile

```diff
@@ -310,7 +310,7 @@ test-envoy-integ: $(ENVOY_INTEG_DEPS)
 	@$(SHELL) $(CURDIR)/test/integration/connect/envoy/run-tests.sh
 
 proto:
-	protoc agent/connect/ca/plugin/*.proto --gofast_out=plugins=grpc:../../..
+	protoc agent/connect/ca/plugin/*.proto --gofast_out=plugins=grpc,Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types:../../..
```

Comment:
Just checking because I've never seen this before is `Mgoogle/` def not a typo? 😄

Reply:
It's definitely not. See https://github.com/gogo/grpc-example.